rack-throttle.gemspec

```diff
@@ -35,6 +35,8 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'rspec'
   gem.add_development_dependency 'yard'
   gem.add_development_dependency 'timecop'
+  gem.add_development_dependency 'pry'
+  gem.add_development_dependency 'awesome_print'
```

Comment:
Are those gems _really_ required for development of this gem?

Reply:
no they are not, forgot to remove it again .-)